mPowerSDK/Startup/APHAppDelegate.h

```diff
@@ -46,7 +45,7 @@
 @property  (nonatomic, readonly) NSInteger environment;
 @property  (nonatomic, readonly) NSArray <APCTaskReminder *> * _Nonnull allTaskReminders;
 @property  (nonatomic, readonly) NSDictionary * _Nonnull appearanceInfo;
-@property  (nonatomic, strong)  APHProfileExtender * _Nullable profileExtender;
+@property  (nonatomic, strong)  APHProfileExtender * _Nonnull profileExtender;
```

Comment:
Turns out it was just how I defined it originally in the fix I'd put in for the same autorelease issue.